libs/ballot-interpreter/src/legacy_adapter.ts

```diff
@@ -77,9 +81,23 @@ function convertNewHampshireNextMarkToSharedMark(
   };
 
   if (option.type === 'candidate' || option.type === 'yesno') {
+    const score =
+      // use the mark fill score if the bubble is filled in enough
+      scoredMark.fillScore >= markThresholds.definite
+        ? scoredMark.fillScore
+        : // or, if there's enough text in the write-in area, score that at 100%
+        scoredWriteInArea &&
+          typeof markThresholds.writeInTextArea === 'number' &&
+          scoredWriteInArea.score >= markThresholds.writeInTextArea
+        ? 1
+        : // otherwise, just leave the mark fill score alone
+          scoredMark.fillScore;
+    if (scoredWriteInArea) {
+      console.log({ scoredMark, scoredWriteInArea, score });
```

Comment:
Is this meant to be left in? Maybe convert to `debug`?

Reply:
Nope, will remove.